empress/support_files/js/colorer.js

```diff
@@ -553,6 +565,8 @@ define(["chroma", "underscore", "util"], function (chroma, _, util) {
         "#808000",
         "#008080",
     ];
+    Colorer.__GR_OR_PR = "GnOrPr";
```

Comment:
```suggestion
    Colorer.__GN_OR_PR = "GnOrPr";
```

Just to make this consistent with `Colorer.__gnOrPr` -- if we use different abbreviations i suspect it'll cause problems down the line